packages/php/woorelease-extension/src/Utils/Git.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Git class.
+ *
+ * @package Automattic\WooCommerce\Grow\WR\Utils
+ */
+
+namespace Automattic\WooCommerce\Grow\WR\Utils;
+
+use WR\Tools\Utils;
+
+/**
+ * Class Git
+ *
+ * Extends the Git class from the woorelease package with additional functionality.
+ */
+class Git extends \WR\Tools\Git {
+
+	/**
+	 * Checks if a branch exists in a remote repository.
+	 *
+	 * @param string $repository_url The remote repository URL.
+	 * @param string $branch         The branch name.
+	 *
+	 * @return bool True if the branch exists, false otherwise.
+	 */
+	public static function is_branch_exists( $repository_url, $branch ) {
```

Comment:
Changing the name helps this to read better.

```suggestion
	public static function does_branch_exist( $repository_url, $branch ) {
```